src/ram/transform/MakeIndex.h

```diff
@@ -74,17 +74,23 @@ class MakeIndexTransformer : public RamTransformer {
      */
     using ExpressionPair = std::pair<std::unique_ptr<RamExpression>, std::unique_ptr<RamExpression>>;
 
+    ExpressionPair getSignedExpressionPair(const RamConstraint* binRelOp, size_t& element, int identifier);
+    ExpressionPair getUnsignedExpressionPair(const RamConstraint* binRelOp, size_t& element, int identifier);
+    ExpressionPair getFloatExpressionPair(const RamConstraint* binRelOp, size_t& element, int identifier);
+
     ExpressionPair getLowerUpperExpression(RamCondition* c, size_t& element, int level);
 
     /**
      * @brief Construct query patterns for an indexable operation
+     * @param Attribute types to indicate type of each attribute in the relation
```

Comment:
This comment is not set up correctly for doxygen. `@param` needs the variable name (and, as above, the @brief is not needed).